pipelines/deployer/deployer.yaml

```diff
@@ -731,10 +731,39 @@ jobs:
                   echo "done"
               fi
           fi
+  - task: generate-terraform-var-overrides
+    image: task-toolbox
+    timeout: 40s
+    params:
+      CLUSTER_NAME: ((cluster-name))
+      ACCOUNT_ROLE_ARN: ((account-role-arn))
+      AWS_REGION: eu-west-2
+      AWS_DEFAULT_REGION: eu-west-2
+    config:
+      platform: linux
+      run:
+        path: /bin/bash
+        args:
+        - -euo
+        - pipefail
+        - -c
+        - |
+          echo "assuming aws deployer role..."
+          AWS_CREDS="$(aws-assume-role $ACCOUNT_ROLE_ARN)"
+          eval "${AWS_CREDS}"
+          # Look for ASGs for this cluster that have a single AZ (i.e. worker ASGs) and make a map of their AZs to desired counts
```

Comment:
Perhaps we should add a tag to the ASGs so we're not relying on "they happen to have 1 AZ associated"..?